src/PrestaShopBundle/Resources/config/services/adapter/alias.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  _defaults:
+    public: true
+
+  PrestaShop\PrestaShop\Adapter\Alias\Validate\AliasValidator:
+    autowire: true
+    public: false
+
+  PrestaShop\PrestaShop\Adapter\Alias\Repository\AliasRepository:
```

Comment:
it should be autowired and public: false

Reply:
still relevant. It can inherit defaults (the autowire and public false) and you should be able to remove arguments from the definition.